code/modules/blob/blob.dm

```diff
@@ -54,7 +54,7 @@
 	update_icon()
 
 /obj/effect/blob/proc/expand(var/turf/T)
-	if(istype(T, /turf/unsimulated/) || istype(T, /turf/space))
+	if(istype(T, /turf/unsimulated/) || istype(T, /turf/space) || istype(T, /turf/unsimulated/mineral))
```

Comment:
Shouldn't `istype(T, /turf/unsimulated/)` return true for `/turf/unsimulated/mineral` anyway?